src/main.c

```diff
@@ -1,18 +1,20 @@
 #include <stdlib.h>
+#include <string.h>
 #include "bmp.h"
 
+typedef unsigned long long PIXEL;
+ 
 int main() {
-	int h = 1080;
-	int w = 1920;
-
-	char *bmp = malloc(h * w * 3);
-	for (int index = 0; index < h * w * 3; index = index + 3){
-		bmp[index] = 0x00;
-		bmp[index + 1] = 0x00;
-		bmp[index + 2] = 0xff;
+	PIXEL h = 4000;
+	PIXEL w = 4000;
+	PIXEL datasize = h * w * 3;
+	char *bmp = malloc(datasize);
+	
+	// Quit if memory is not available
+	if (bmp == NULL){
+		return 1;
 	}
-    write_bmp("pixel.bmp", bmp, h * w * 3 , w);
-}
-
 
-	
\ No newline at end of file
+	ismandelbrot(h, w, bmp);

```

Comment:
I would suggest renaming this function. You pick a name, but is_mandelbrot seems to hint that the process checks if something is Mandelbrot and returns a bool, when indeed what happens is that the function calculates a mandelbrot fractal frame based on some world params and populates an array of memory.